### ISSUE TITLE
When calling app.bsky.feed.getActorFeeds a cursor is returned even though the feed list cannot be paginated

### ISSUE THREAD

**Issue by jasonculverhouse** [bluesky-social/atproto#1176]
**Describe the bug**

When calling app.bsky.feed.getActorFeeds a cursor is returned even though the feed list cannot be paginated

**To Reproduce**

1. `curl 'https://bsky.social/xrpc/app.bsky.feed.getActorFeeds?actor=did:plc:cndfx4udwgvpjaakvxvh7wm5&limit=30'`

request the feed (only 2 are present) yet a cursor is returned.  

```
{
  "cursor": "1686067091549::bafyreibe55je67svx26xsff76hlhn65zcpyask6rsoi6x5kok6sufxabdy",
  "feeds": [
    {
      "uri": "at://did:plc:cndfx4udwgvpjaakvxvh7wm5/app.bsky.feed.generator/flipboard-tech",
      "cid": "bafyreifnjzcl577uhqc7buftpym4feiul6ry5knigymrtm6f4xukpnvdui",
      "did": "did:web:bsky.flipboard.com",
      "creator": {
        "did": "did:plc:cndfx4udwgvpjaakvxvh7wm5",
        "handle": "flipeng.bsky.social",
        "viewer": {
          "muted": false,
          "blockedBy": false,
          "following": "at://did:plc:d4yzg3vadl536rkihgygkvv6/app.bsky.graph.follow/3juwgbo6mvv27",
          "followedBy": "at://did:plc:cndfx4udwgvpjaakvxvh7wm5/app.bsky.graph.follow/3jvfxutaw562t"
        },
        "labels": []
      },
      "displayName": "Flipboards's Tech feed",
      "description": "Curated by Flipboard's state of the art topics recommendation system.",
      "avatar": "https://cdn.bsky.social/imgproxy/7pOnAO_vkyST0Rw5bSuB3jvdyOz4jFQL0HJ6DS3QZMI/rs:fill:1000:1000:1:0/plain/bafkreigbeq7c4dvrlpoyhnvlitrqs56qroafwgkmy6kx6mo3aun7skskym@jpeg",
      "likeCount": 11,
      "viewer": {},
      "indexedAt": "2023-06-06T20:12:22.029Z"
    },
    {
      "uri": "at://did:plc:cndfx4udwgvpjaakvxvh7wm5/app.bsky.feed.generator/Flipboard-tech",
      "cid": "bafyreibe55je67svx26xsff76hlhn65zcpyask6rsoi6x5kok6sufxabdy",
      "did": "did:web:bsky.flipboard.com",
      "creator": {
        "did": "did:plc:cndfx4udwgvpjaakvxvh7wm5",
        "handle": "flipeng.bsky.social",
        "viewer": {
          "muted": false,
          "blockedBy": false,
          "following": "at://did:plc:d4yzg3vadl536rkihgygkvv6/app.bsky.graph.follow/3juwgbo6mvv27",
          "followedBy": "at://did:plc:cndfx4udwgvpjaakvxvh7wm5/app.bsky.graph.follow/3jvfxutaw562t"
        },
        "labels": []
      },
      "displayName": "Flipboards's Tech feed",
      "description": "Curated by Flipboard's state of the art topics recommendation system.",
      "avatar": "https://cdn.bsky.social/imgproxy/7pOnAO_vkyST0Rw5bSuB3jvdyOz4jFQL0HJ6DS3QZMI/rs:fill:1000:1000:1:0/plain/bafkreigbeq7c4dvrlpoyhnvlitrqs56qroafwgkmy6kx6mo3aun7skskym@jpeg",
      "likeCount": 0,
      "viewer": {},
      "indexedAt": "2023-06-06T19:58:11.728Z"
    }
  ]
}

```

**Expected behavior**

no `cursor` is present in the response if there are no more results.

**Details**

 - Operating system:
 - Node version:

**Additional context**

**Comment by jasonculverhouse** [comment]
You can repo this in the web app If you just click back and forth on the custom feeds it keeps adding feeds again and again  

![image](https://github.com/bluesky-social/atproto/assets/286258/69797a5c-ba10-41ac-84b2-7702a3e3752b)

**Comment by devinivy** [comment]
Thanks for the report!  This looks like an issue with normalizing the `createdAt` times during indexing.  The cursor is considered accurate, but that result should be omitted from the next page rather than included.  Have a PR up with the fix here: #1178.
